### DISCUSSION TITLE
vc-primitive-particled第一次渲染时颜色为什么给不上

### DISCUSSION THREAD

**Discussion by Aleeeen** [zouyaoji/vue-cesium#415] [Q&A]
demo上的烟花特效就是白色的

**Comment by zouyaoji** [comment]
- -记不得了，我这边给上了哈。

**Reply by Aleeeen** [reply]
准确来说是初始化时startColor和endColor没生效，如果同时给了color，startColor和endColor，则是使用的color指定的颜色，如果只设置了startColor和endColor则是白色。在渲染后，再改变startColor和endColor才会使startColor和endColor生效。
(图片传不上来，麻烦看下下面的代码，初始化时给的黄黑没生效，点击加载改变startColor和endColor才生效)

[demo.txt](https://github.com/zouyaoji/vue-cesium/files/12237543/demo.txt)

**Reply by zouyaoji** [reply]
收到，空了看看哈。
